tests/ci/integration/run_curl_integration.sh

```diff
@@ -30,14 +30,14 @@ cd ${SCRATCH_FOLDER}
 
 function curl_build() {
   autoreconf -fi
-  ./configure --enable-warnings --enable-werror --with-openssl=${AWS_LC_INSTALL_FOLDER}
-  make -j ${NUM_CPU_THREADS}
-  make -j ${NUM_CPU_THREADS} examples
+  ./configure --enable-warnings --enable-werror --with-openssl=${AWS_LC_INSTALL_FOLDER} --without-libpsl
+  make -j "$NUM_CPU_THREADS"
+  make -j "$NUM_CPU_THREADS" examples
```

Comment:
we're not building `.so`s for AWS-LC, so we can't use the `ldd` validation we use elsewhere to assert that the project under test is built properly against AWS-LC.

the `curl` distribution on my ubuntu 22.04 box shows the OPENSSL_VERSION string in its `--version` output. perhaps we could grep for AWS-LC there?

```
$ curl --version
curl 7.81.0 (x86_64-pc-linux-gnu) libcurl/7.81.0 OpenSSL/3.0.2 zlib/1.2.11 brotli/1.0.9 zstd/1.4.8 libidn2/2.3.2 libpsl/0.21.0 (+libidn2/2.3.2) libssh/0.9.6/openssl/zlib nghttp2/1.43.0 librtmp/2.3 OpenLDAP/2.5.14
Release-Date: 2022-01-05
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtmp rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS brotli GSS-API HSTS HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz NTLM NTLM_WB PSL SPNEGO SSL TLS-SRP UnixSockets zstd
```

Reply:
Will do in a follow up PR